src/sentry/static/sentry/app/routes.jsx

```diff
@@ -422,7 +427,7 @@ function routes() {
           />
         </Route>
       </Route>
-
+      {hooksPreOrgRoutes}
```

Comment:
Let's move this to line 356, and call it `routes:start` or `routes:begin` since the hook isn't really tied to organization, the name should be more related to its location in the root Route tree. And I can see the need in the future for routes early/at the start of the tree.